changelog.d/14594.doc

```diff
@@ -0,0 +1 @@
+Add Mastodon and Fediverse Single-Sign On setup instructions for Synapse.
```

Comment:
Wikipedia [defines](https://en.wikipedia.org/w/index.php?title=Fediverse&oldid=1124407313) the fediverse as

> an ensemble of [federated](https://en.wikipedia.org/wiki/Federation_(information_technology)) (i.e. interconnected) servers that are used for [web publishing](https://en.wikipedia.org/wiki/Web_publishing) (i.e. [social networking](https://en.wikipedia.org/wiki/Social_networking), [microblogging](https://en.wikipedia.org/wiki/Microblogging), [blogging](https://en.wikipedia.org/wiki/Blogging), or [websites](https://en.wikipedia.org/wiki/Websites)) and file hosting, but which, while independently hosted, can communicate with each other.

and lists a variety of software other than Mastodon.

```suggestion
Add Single-Sign On setup instructions for Mastodon instances.
```

Reply:
Yeah. Fediverse is way too broad of a term. However, would the better term be `Mastodon-based` ? Since there are several forks based on Mastodon, like [Pleroma](https://pleroma.social/). The one I'm using with Synapse is using [Soapbox](https://gitlab.com/soapbox-pub/soapbox) frontend.

Reply:
"Mastodon-based" sounds fine :+1: